### PR TITLE
a few more links to errors in the doc

### PR DESCRIPTION
**Pull request by fchapoton**
This adds a few more links from Errors to standard python documentation.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about. We're here to help! -->

### CODE REVIEW

File: src/sage/modular/abvar/abvar.py [hunk 1]
```diff
@@ -3235,16 +3238,14 @@ def isogeny_number(self, none_if_not_known=False):
...
         Return the number (starting at 0) of the isogeny class of new
         simple abelian varieties that self is in. If self is not simple,
-        raises a ValueError exception.
+        raises a :class:`ValueError` exception.
 
         INPUT:
 
-
-        -  ``none_if_not_known`` - bool (default: False); if
+        -  ``none_if_not_known`` -- bool (default: False); if
```
Comment (dcoudert) [inline src/sage/modular/abvar/abvar.py]: ` ``False`` `?

File: src/sage/matrix/matrix_integer_dense.pyx [hunk 2]
```diff
@@ -1712,8 +1712,8 @@ cdef class Matrix_integer_dense(Matrix_dense):
         Return a pair (F, C) such that the rows of C form a symplectic
         basis for self and ``F = C * self * C.transpose()``.
 
-        Raise a ValueError if self is not anti-symmetric, or self is not
-        alternating.
+        Raise a :class:`ValueError` if self is not anti-symmetric,
```
Comment (dcoudert) [inline src/sage/matrix/matrix_integer_dense.pyx]: ` ``self`` `?

File: src/sage/modular/abvar/abvar.py [hunk 3]
```diff
@@ -580,9 +580,11 @@ def newform_decomposition(self, names=None):
     def newform_label(self):
         """
```
Comment (dcoudert) [inline src/sage/modular/abvar/abvar.py:583]: May be not in this PR, but many `"""` should be changed to `r"""`.

**Review by dcoudert — CHANGES_REQUESTED** [review]

File: src/sage/modular/abvar/abvar.py (hunk 3 above)
Comment (fchapoton) [reply src/sage/modular/abvar/abvar.py:583]: not needed here as there is no keyword in the latex code

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**